spec/rails_helper.rb

```diff
@@ -75,6 +75,7 @@
   "selenium-release.storage.googleapis.com",
   "developer.microsoft.com/en-us/microsoft-edge/tools/webdriver",
   "api.knapsackpro.com",
+  ENV.fetch("CHROME_URL", nil),
```

Comment:
is nil safe to have in this array? or should this be a conditional array append?

Reply:
yeah, we want `nil` here because the code within `spec/support/initializers/cuprite.rb` rely on its existence to determine the proper chromedriver setup.